content/english/anti-vegan/personal-choice.md

```diff
@@ -9,42 +9,48 @@ weight: 5
 
 # Argument
 
-> What we eat is a personal choice, it's not right for you to go round telling people what to do.
+> What we eat is a personal choice and it's not right for you to go round telling people what to do.
 
 or
 
-> It's my choice, why can't you respect that?
+> It's my choice. Why can't you respect that?
 
 # Rebuttal  
 
 ## Quick Response
 
-Personal choices cease to be purely personal when they harm others.  
+> Yes, it's a personal choice. However, where that choice causes unnecessary harm I am morally obligated to speak out
+
+or
+
+> Yes, it's a personal choice but no choice that causes unnecessary harm should be respected.
 
 ## Detail - Analytic
 
-The core idea underpinning the concept "It's my personal choice" is that we have the right to choose for ourselves the correct course of action and that the choice, once made, should be respected. 
+> A _golden rule_ of moral philosophy is; "all things being equal you ought to choose the action that does the least harm” 
+
+Anyone who accepts:
+
+1. It is not a necessary need to eat meat.
+2. Eating meat is the indirect cause of great harm.
+3. The golden rule that, "all things being equal you ought to choose the action that does the least harm”
 
-The hidden premises of respecting some-one else's choice go something like:
+will necessarily conclude:
 
-1. Everyone is allowed their own opinion even if you don't agree with it 
-2. My opinion informs my actions through choice
-3. My actions, choices, and opinions should all be afforded the same level of acceptance
+- Your _personal choice_ to eat meat causes unnecessary harm, does not warrant respect, and morally obligates me to speak out.
 
-This takes a freedom of thought/speech type model and applies it to actions under the guise of 'choice'. And it works intuitively because there are scenarios in which this makes sense!
+ 
+> Premise 1 is a factual claim. If you doubt its veracity please read [A well planned vegan diet is suitable for people of all ages.] (/en/facts/suitable-for-all/)
 
-- It's my choice to drop out of uni
-- It's my choice to dye my hair
-- It's my choice to spend all my money on a holiday instead of saving it for a house
+> Premise 2 is the factual claim that animals are harmed by commercial farming and slaughter. See [Animal slaughter methods] (/en/facts/animal-slaughter-methods/)
```

Comment:
These links might work in Hugo, but won't work in the app unfortunately. They need to be of the format 
```
[Animal slaught methods]({{<ref "/facts/animal-slaughter-methods">}})
```